web/src/components/ProjectInfo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ProjectWrapper = styled.div`
+  ${tw`container flex flex-col text-brand-9 lg:flex-row mb-24`}
+`;
+
+const StyledHeading = styled.div`
+  ${tw`font-bold mr-16 mb-4 text-5xl lg:text-6xl`}
+`;
+
+const StyledBody = styled.div`
+  ${tw`w-auto	text-2xl`}
```

Comment:
Should probably add a width class for larger screens. `lg:w-2/3`. You can probably remove the `w-auto` class too.

Reply:
@asasmith i had w 2/3 but it would look weird on mobile. I thought auto looked better across breakpoints.